Sources/Public/Animation/LottieAnimationLayer.swift

```diff
@@ -312,6 +312,33 @@ public class LottieAnimationLayer: CALayer {
     }
   }
 
+  public override var anchorPoint: CGPoint {
+    get {
+      if let animationLayer = animationLayer {
```

Comment:
Instead of modifying the optional `animationLayer` we should use the non-optional `lottieAnimationLayer`.